neuraxle/base.py

```diff
@@ -1410,6 +1438,17 @@ def fit(self, data_inputs, expected_outputs=None) -> 'NonFittableMixin':
 class NonTransformableMixin:
     """
     A pipeline step that has no effect at all but to return the same data without changes.
+    Transform method is automatically implemented as changing nothing.
+
+    Example :
+    .. code-block:: python
+        class PrintOnFit(NonTransformableMixin, BaseStep):
+            def __init__(self):
+                BaseStep.__init__(self)
+
+            def fit(self, data_inputs, expected_outputs=None) -> 'FitCallbackStep':
+                print((data_inputs, expected_outputs))
+                return self
```

Comment:
This could be cool to extract to a /examples/*.py file and like to it.

Reply:
created an issue #126